Permanent/ViewControllers/WelcomePageViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  WelcomePageViewController.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 06.07.2021.
+//
+
+import UIKit
+
+class WelcomePageViewController: BaseViewController<AuthViewModel>  {
+    
+    var archiveName: String?
+    
+    @IBOutlet weak var welcomePageView: UIView!
+    @IBOutlet weak var pageImage: UIImageView!
+    @IBOutlet weak var primaryLabelField: UILabel!
+    @IBOutlet weak var secondaryLabelField: UILabel!
+    @IBOutlet weak var acceptButton: RoundedButton!
+    @IBOutlet weak var closeButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        initUI()
+    }
+    
+    @IBAction func acceptButton(_ sender: Any) {
+        closePopUp()
+    }
+    
+    @IBAction func closeButton(_ sender: Any) {
+        closePopUp()
+    }
+    
+    func initUI() {
+        view.backgroundColor = UIColor.black.withAlphaComponent(0.3)
+        
+        welcomePageView.backgroundColor = .darkBlue
+        welcomePageView.clipsToBounds = true
+        welcomePageView.layer.cornerRadius = 10
+        
+        primaryLabelField.textColor = .white
+        primaryLabelField.font = Text.style2.font
+        primaryLabelField.numberOfLines = 2
+        primaryLabelField.text = "Welcome to The \(archiveName ?? "name") Archive".localized()
```

Comment:
We won't be able to localize this string as it is now, because the archive name is dynamic. 
I suggest we do something like: "Welcome to The <ARCHIVE_NAME> Archive".localized().replacing(occurencesOf: "<ARCHIVE_NAME>", with: archiveName)